docs/src/code.js

```diff
@@ -120,17 +119,19 @@ function overlap(x, y) {
 
 function stars() {
     const stars = (_viewport.worldWidth * _viewport.worldHeight) / Math.pow(STAR_SIZE, 2) * 0.1
+    console.log('stars', stars);
```

Comment:
this should be removed?

Reply:
Seems so... They are from the initial commits from @Aeolun which I just moved onto this PR so I could add the commits which made it usable for me...